magicctapipe/io/tests/test_io.py

```diff
@@ -262,9 +262,9 @@ def test_load_train_data_files_g(gamma_stereo):
     """
 
     events = load_train_data_files(str(gamma_stereo[0]))
-    assert list(events.keys()) == ["LST1_M1", "LST1_M2", "LST1_M1_M2"]
+    assert list(events.keys()) == ["LST1_M1", "LST1_M1_M2", "LST1_M2"]
```

Comment:
test is failing:

```python
def test_load_train_data_files_g(gamma_stereo):
        """
        Check dictionary
        """
    
        events = load_train_data_files(str(gamma_stereo[0]))
>       assert list(events.keys()) == ["LST1_M1", "LST1_M1_M2", "LST1_M2"]
E       AssertionError: assert ['LST1_M1', '..._M2', 'M1_M2'] == ['LST1_M1', '...2', 'LST1_M2']
E         Left contains one more item: 'M1_M2'
E         Full diff:
E         - ['LST1_M1', 'LST1_M1_M2', 'LST1_M2']
E         + ['LST1_M1', 'LST1_M1_M2', 'LST1_M2', 'M1_M2']
E         ?                                    +++++++++
```

Reply:
this is marked by github as outdated, and currently all the CI tests are passing. resolving this comment